xapi/tipb/schema.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto2";
+
+package tipb;
+
+option java_multiple_files = true;
+option java_package = "com.pingcap.tidb.tipb";
+option java_outer_classname = "SelectProto";
```

Comment:
According to https://developers.google.com/protocol-buffers/docs/javatutorial#why-use-protocol-buffers
This option can be absent, then the class name would be camel case file name.
I'll remove it.